src/app/src/api/boundaries.js

```diff
@@ -27,11 +27,47 @@ const boundaryApi = api.injectEndpoints({
         }),
 
         startNewBoundary: build.mutation({
-            query: newBoundary => ({
-                url: '/boundaries/',
-                method: 'POST',
-                data: newBoundary,
-            }),
+            query: ({ reference_images, shape, ...details }) => {
+                const formData = new FormData();
```

Comment:
The reason we're sending `FormData` rather than JSON is because of the file uploads?

Reply:
Yes.

Also regarding:

> Can this reuse any of the reference image upload work done in https://github.com/azavea/iow-boundary-tool/pull/144? In lieu of the FormData parsing?

#144 did not implement file uploading. It uploads just the reference image meta data.